db/migrations/20150809163538-links.js

```diff
@@ -9,6 +9,14 @@ export function up(queryInterface, Sequelize) {
       type: Sequelize.STRING,
       allowNull: false,
     },
+    goDescription: {
```

Comment:
You will want to make a new migration file that specifically adds these columns. Updating existing migration files is an anti-pattern.
Also, naming: `description` and `public`